tools/package.json

```diff
@@ -101,9 +101,12 @@
     "@types/source-map-support": "^0.5.4",
     "@types/uuid": "^3.4.4",
     "@types/ws": "^7.4.7",
+    "@typescript-eslint/eslint-plugin": "^5.15.0",
+    "@typescript-eslint/parser": "^5.15.0",
     "babel-preset-expo": "^9.0.2",
+    "eslint": "^8.11.0",
     "expo-module-scripts": "^2.0.0",
-    "prettier": "^2.3.1",
+    "prettier": "^2.6.0",
```

Comment:
IIRC `@typescript-eslint` dependencies should not be required to install manually, since `tools` are based on `expo-module-scripts`, which depends on `eslint-config-universe`, and this package list the `@typescript-eslint` packages in direct dependencies:
* https://github.com/expo/expo/blob/main/packages/eslint-config-universe/package.json#L45-L46

It looks like the source of this might be the Readme file, which seems to include those packages in the recommended install command, however looking at `eslint-config-universe` package.json history it looks like `@typescript-eslint` packages have been listed in the direct `dependencies` for at least last 2 years, so I'm not sure why they were added to the recommended install command in the first place. 😅

Reply:
I think it might have just been a missing eslint or something, but I'll try to narrow it down.